.github/workflows/build.yaml

```diff
@@ -0,0 +1,38 @@
+# Build Windows workflow
+name: build
+
+on:
+  push:
+  workflow_dispatch:
+
+jobs:
+  build-windows:
+    runs-on: windows-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2.3.4
+      - name: Setup Python
+        uses: actions/setup-python@v2.2.2
+        with:
+          python-version: 3.9
+      - name: Install dependencies
+        run: |
+          python -m pip install --upgrade pip
+          pip install -r requirements.txt
+      - name: Build Windows
+        run: pyinstaller --clean --noconfirm specs/windows-terminservice.spec --add-data "tools/cloudscraper;./cloudscraper/"
```

Comment:
bitte den --add-data command in die .spec erstellung hinzufügen. (und in der Readme ändern). Damit kann dann auch der python code aus der .spec raus. Einfach nocheinmal die .spec mit dem --add-data command für tools/cloudscraper ausführen. (Linux und Win)

Reply:
Sorry, kannst du das nochmal ausführen, ich verstehe nicht ganz was du meinst. Pyinstaller wird hier ja schon mit `tools/cloudscraper` ausgeführt, was genau soll hier noch geändert werden?